filter_test.go

```diff
@@ -169,3 +169,12 @@ func TestConvertFilter(t *testing.T) {
 		}
 	}
 }
+
+func TestInFilter(t *testing.T) {
+	eleven, _ := NewManifest("testdata/k-s-v0.11.0.yaml")
+	twelve, _ := NewManifest("testdata/k-s-v0.12.1.yaml")
+	new := twelve.Filter(None(In(eleven)))
+	if len(new.Resources()) != 1 {
+		t.Error("Missing the autoscaler-hpa")
+	}
```

Comment:
This PR will address the issue I have. 
Do I understand correctly?
`twelve.Filter(None(In(eleven)))` means locating the resources exist in `twelve`, but not in `eleven`.
`twelve.Filter(In(eleven))` means locating the resources exist both in `twelve` and `eleven`.

Reply:
Yes, exactly. It's essentially the *intersection* of the two. Makes me wonder whether `Intersect` is a better name than `In`, though I like the conciseness of this when finding which resources are now obsolete:
```go
  deletes := v1.Filter(Not(In(v2)))
```

Reply:
Cool, we got it.